eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -130,6 +130,50 @@ public function testCreateLocation()
         ];
     }
 
+    /**
+     * Test for the createLocation() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::createLocation()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testCreateLocation
+     * @depends eZ\Publish\API\Repository\Tests\ContentServiceTest::testHideContent
+     */
+    public function testCreateLocationChecksContentVisibility()
```

Comment:
CS
```suggestion
    public function testCreateLocationChecksContentVisibility(): void
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3066/commits/c5536b0a3ec34bbdc63a99ecd01b0cbb74705187